packages/api/resolvers/types/product-variation-assignment-vector.js

```diff
@@ -3,7 +3,7 @@ export default {
     return `${product._id}:${key}=${value}}`;
   },
   option(obj) {
-    return obj.product.variation(obj.key).optionObject(obj.value);
+    return obj.product.variation(obj.key)?.optionObject(obj.value);
```

Comment:
Why that?

Reply:
it'll throw an internal server error when there are no variations and you try to view a configurable product variation in the control panel.